resources/dicts/relationship_events/normal_interactions/dislike/increase.json

```
@@ -304,10 +304,10 @@
 	},
 	{
 		"id": "dislike_dep_leader_inc_high1",
-        "intensity": "high",
+        	"intensity": "high",
 		"interactions": [
-            "m_c pulls r_c aside and reminds {PRONOUN/r_c/object} {PRONOUN/m_c/subject} could easily exile {PRONOUN/r_c/object}.",
-            "m_c scolded r_c for not doing the hard task {PRONOUN/m_c/subject} assigned {PRONOUN/r_c/object} despite knowing it was a job for more than one cat.",
+            		"m_c pulls r_c aside and reminds {PRONOUN/r_c/object} {PRONOUN/m_c/subject} could easily exile {PRONOUN/r_c/object}.",
```

```suggestion
            		"m_c pulls r_c aside and reminds {PRONOUN/r_c/object} m_c could easily exile {PRONOUN/r_c/object}.",
```
just to help clarify the sentence in case both cats have similar pronouns